src/password.cpp

```diff
@@ -15,7 +16,7 @@ QString Password::cryptPassword(const QString &password) const
     QByteArray salt("$6$");
     int len = alpha.count();
     for (int i = 0; i < 16; i++)
-        salt.append(alpha.at((qrand() % len)));
+        salt.append(alpha.at((QRandomGenerator::global()->bounded(len))));
```

Comment:
@omaemae There is a pair of extra brackets 。。。。。

Reply:
@ChungZH Fixed in upstream now. :)